### ISSUE TITLE
How do I prepare phenotype data for analyzing binary traits?

### ISSUE THREAD

**Issue by jeong2624** [jiabowang/GAPIT#107]
Hello!

I'm planning to use GAPIT for GWAS data analysis. However, my phenotype data is categorical, while the models provided by GAPIT are based on numerical data.

The GAPIT manual mentions that GAPIT can analyze binary traits (case-control), but it doesn't explain how explicitly.

If I want to analyze binary traits using GAPIT, should I encode them as 0 and 1 in the phenotype data?

Also, if the phenotype data consists of nominal or categorical values, could you please explain how to analyze it with GAPIT?

Lastly, can you confirm if all GAPIT models are available for this scenario?

**Comment by jiabowang** [comment]
Yes, the GAPIT works for binary traits and continuous traits. But in the binary traits, the heritability is always low. So the detectability of GWAS always shows no-enough. That needs many more individuals.

**Comment by jeong2624** [comment]
Hello! I am confused for GAPIT.

As mentioned, GAPIT is designed for both binary and continuous traits. 
According to page 48 of the GAPIT User Manual, the answer to whether it is appropriate to analyze binary traits (case-control) is 'Yes, there are many applications.' 

In this case, should I only use the GLM model, or can I also apply other models such as MLM, CMLM, FarmCPU, BLINK, and MLMM?"

To my knowledge, while most GWAS models were originally developed for quantitative traits, GLM can handle binary traits by incorporating logistic regression.
